roles/uperf-scale/tasks/start_client.yml

```diff
@@ -0,0 +1,63 @@
+---
+
+- name: Get pod info
+  k8s_facts:
+    kind: Pod
+    api_version: v1
+    namespace: '{{ operator_namespace }}'
+    label_selectors:
+      - type = {{ ansible_operator_meta.name }}-bench-server-{{ trunc_uuid }}
+  register: server_pods
+
+- name: set pod_sequence
+  set_fact:
+    pod_sequence: "{{ (num_server_pods|int - 1) if (workload_args.serviceip|default(False) == True and workload_args.servicetype | default('clusterip') == 'nodeport') else (pod_hi_idx|int) }}"
```

Comment:
Looks like this change was rolled in together with a few others. It look pretty dense here. Hope it works as intended.